client/data/authorizations/resolvers.ts

```diff
@@ -64,6 +72,49 @@ export function* getAuthorizations( query: Query ): any {
 	yield updateAuthorizations( query, getMockedRows() );
 }
 
+export function* getAuthorization( id: string ): Generator< unknown > {
+	// TODO replace mocked implementation when server is ready
+	const randomAmount = () => {
+		return 1400 + Math.floor( Math.random() * 5000 );
+	};
+
+	const randomDate = dateI18n(
+		'M j, Y / g:iA',
+		moment.utc( new Date() ).local().toISOString()
+	);
+
+	const randomCaptureDate = dateI18n(
```

Comment:
Is the time part necessary here? The design screenshot in #4506 doesn't have it.

Also, when the data is eventually wired up with server, would there be a benefit to have this string in `ISO-8601` format?